src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -405,6 +409,7 @@ def generate_config(
 
         alertmanager_user, alertmanager_password = self.mgr._get_alertmanager_credentials()
         prometheus_user, prometheus_password = self.mgr._get_prometheus_credentials()
+        FSID = self.mgr.get('config')['fsid']
```

Comment:
minor point, we tend to access this with `self.mgr._cluster_fsid`, just to keep it consistent in the codebase.

Reply:
Thanks @adk3798 , Updated the PR